Source/Message/MessageClient.swift

```diff
@@ -21,10 +21,22 @@
 import Foundation
 
 /// Message HTTP client.
-open class MessageClient: CompletionHandlerType<Message> {
+public class MessageClient {
+    
+    /// Alias for closure to handle a service response along with a Message object.
+    public typealias ObjectHandler = (ServiceResponse<Message>) -> Void
```

Comment:
Should we need to define the public typealias in every XXXClient class?  Is it more straightforward and clear with (ServiceResponse<XXX>) -> Void directly in the function signature?

```
func list(roomId: String, before: String? = nil, beforeMessage: String? = nil, max: Int? = nil, queue: DispatchQueue? = nil, completionHandler: @escaping (ServiceResponse<[Message]>) -> Void)
```